### PR TITLE
[GENEVA-2018] added stub for 2018 geneva event

### PR DESCRIPTION
**Pull request by matemaz**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2018] Add Bluth Company as a sponsor`*

### CODE REVIEW

**Comment by bridgetkromhout** [comment]
Hi, @matemaz - this is a good start. You are missing the YAML config file, which is necessary.

Two options:

1) use the script which generates it: https://github.com/devopsdays/devopsdays-web/blob/master/utilities/README.md#events

or

2) Manually create and add the necessary edits of the file at https://github.com/devopsdays/devopsdays-web/blob/master/utilities/examples/data/events/yyyy-city.yml to `data/events/2018-geneva.yml`

**Comment by bridgetkromhout** [comment]
Hi, @matemaz - if you have questions, please comment here or email info@devopsdays.org. We could set up the yml file for you but we don't have your organizer info, dates, etc. Thanks.

**Comment by matemaz** [comment]
Hi thanks very much.
I’ll give it one more try just now.
Thanks very much for your support.
Best
Matteo

--
@matemaz
http://mazzeri.net
promoting curiosity

File: content/events/2018-geneva/conduct.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
++++
+date = "2016-10-11T18:48:25+02:00"
+title = "conduct"
+type = "event"
+
++++
+
+## VERSION FRANCAISE
+
+https://www.devopsrex.fr/code-de-conduite/
```
Comment (phrawzty) [inline content/events/2018-geneva/conduct.md]: You're more than welcome to use the CoC from the Paris event, but you might want to copy it for your own purposes - sending people to the website of a _different conference_ will be confusing.

File: content/events/2018-geneva/welcome.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+<!-- <div class = "row">
+  <div class = "col-md-2">
+    <strong>Speakers</strong>
+  </div>
+  <div class = "col-md-8">
+  </div>
+</div> -->
+
+<div class = "row">
```
Comment (phrawzty) [inline content/events/2018-geneva/welcome.md]: Leaving this sections uncommented, but then not filling in the cell, ends up looking like this: 
image

Not a blocker, but you'll want to clean this up in a future PR.

File: content/events/2018-geneva/conduct.md (hunk 1 above)
Comment (matemaz) [reply content/events/2018-geneva/conduct.md]: oops, will change thanks.

File: content/events/2018-geneva/welcome.md (hunk 2 above)
Comment (matemaz) [reply content/events/2018-geneva/welcome.md]: ok will update thanks

**Pull request merged**